src/desktop/apps/gallery_partnerships/client/intercom.js

```diff
@@ -0,0 +1,34 @@
+window.intercomSettings = {
+  app_id: 'q6qkoxl8',
```

Comment:
Let's put this Intercom app id to an environment variable so we can switch without code changes?

Reply:
Sounds good 👍